src/appleseed/renderer/modeling/aov/aov.cpp

```diff
@@ -110,12 +93,12 @@ ColorAOV::ColorAOV(const char* name, const ParamArray& params)
 
 size_t ColorAOV::get_channel_count() const
 {
-    return 3;
+    return 4;
```

Comment:
I am not sure about this change. Most of our color AOVs are 3 channel.
Why did you change it?

Reply:
Because the SadingBuffer is doing 4 * aov_count. So wether we change the shading buffer to use the exact amount of channels which can be difficult or we set all AOV to 4 channels if they are used in the shading buffer.